create_package.sh

```diff
@@ -90,6 +90,7 @@ find /ovms_release/lib/ -iname '*.so*' -exec patchelf --debug --set-rpath '$ORIG
 
 find /opt/intel/openvino/runtime/lib/intel64/ -iname '*.so*' -exec cp -vP {} /ovms_release/lib/ \;
 patchelf --debug --set-rpath '$ORIGIN' /ovms_release/lib/libopenvino.so
+patchelf --debug --set-rpath '$ORIGIN' /ovms_release/lib/libopenvino_tokenizers.so
```

Comment:
don't we need to do the same for libopenvino_genai?

Reply:
Currently it looks ok, no need to change.